e2e/page-objects/boxPage.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable no-unused-expressions */
+const { accessibilityScan, visualScan } = require('../tests/steps/common.steps')
+
+const boxPage = {
+  visitPage(browser) {
+    const url = `${browser.launch_url}/#!/Box`
+
+    browser.url(url)
+    this.waitForElementVisible('@header').expect.element('@backToSpa').to.be.visible
```

Comment:
Why wait for the "backToSpa" element to be visible? Is it not enough to just wait for the "header" to be visible?

Reply:
waiting for header is enough. I've removed backToSpa check because it is redundant.